packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartView/CounterpartView.tsx

```diff
@@ -202,6 +212,7 @@ export const CounterpartView = (props: CounterpartViewProps) => {
             <CounterpartOrganizationView
               actions={actions}
               showCategories={props.showCategories ?? true}
+              isEmailDefault={getDefaultOrganizationContact()}
               counterpart={{
                 taxId: counterpart.tax_id,
                 ...prepareCounterpartOrganization(counterpart.organization),
```

Comment:
I understand your perspective. Are you suggesting that we incorporate this logic into `prepareCounterpartOrganization()` instead? I believe the clarity of a property name can be subjective; while I find `isEmailDefault` intuitive, previous developer have also found it clear, who review the code. It would be helpful to discuss further to reach a consensus on the most intuitive approach for everyone involved

Reply:
I think the `CounterpartOrganisationView` component can decide on its own to show _Default_ or not, without controlling this from above.